src/index.tsx

```diff
@@ -12,10 +12,10 @@ import { ModalHookProvider } from './context/modalContext/ModalContext';
 import { LazyRoute } from './CustomRoutes/LazyRoute';
 import { AuthorizationRoute } from './CustomRoutes/AuthorizedRoute';
 
-const PageNotFound = lazy(
-  () => import(/* webpackChunkName: "PageNotFound" */ './PageNotFound')
-);
-const Me = lazy(() => import(/* webpackChunkName: "Me" */ './Me/Me'));
+// const PageNotFound = lazy(
+//   () => import(/* webpackChunkName: "PageNotFound" */ './PageNotFound')
+// );
+// const Me = lazy(() => import(/* webpackChunkName: "Me" */ './Me/Me'));
```

Comment:
Where `Me` and `PageNotFound` declared now?

Reply:
`<Me>` is at `/pages/me` now.  `<PageNotFound>` isn't anywhere now, but I added a to-do for this; it can be set as the `404` component in Nextjs.